radis/lbl/factory.py

```diff
@@ -1542,6 +1575,77 @@ def _generate_wavenumber_arrays(self):
 
         return
 
+    def predict_time(self):
```

Comment:
I would refactor this as a function of two parameters:
- line optimization : DIT /  LBL 
- broadening algorithm :  FFT / AVP ("Analytical Voigt Profile", i.e. Voigt)

then add a series of 2 nested if/else  ;   and use some NotImplementedError   for any line optimization / convolution   you don't know.

The function "predict_time" will return only one float ; the optimization  will be done elsewhere by calling it several times to compare the results